system/libuv/0001-libuv-port-for-nuttx.patch

```diff
@@ -169,7 +169,24 @@ index 869ae95f..8fabeba5 100644
  }
  
  
-@@ -212,7 +217,7 @@ static void init_threads(void) {
+@@ -188,6 +193,16 @@ static void init_threads(void) {
+   const char* val;
+   uv_sem_t sem;
+ 
++  const uv_thread_options_t params = {
++#ifdef DEF_THREADPOOL_STACKSIZE
```

Comment:
Seems like `DEF_THREADPOOL_STACKSIZE` is always defined, so why do we need to have `ifdef` here?

Reply:
It's defined on NuttX but not on all other platforms

Reply:
yes, but this patch is to build the code with NuttX and not with other platforms :)
Anyway it is minor, so we can leave it as is.

Reply:
We need ensure the same code still can built on other platform and with the result. And we plan to upstream the change to libuv community, so it's important to make the change as general as possible.